README.md

```diff
@@ -1 +1,114 @@
-# motorbike_booking_frontend
\ No newline at end of file
+# Motorbike Booking Front-End
```

Comment:
-  [ ] Your project has no frontend tests.  Kindly add some.

## Explanation

In the real life software industry, for every feature implemented, a test must be added. That way, we know the functionality is not broken as we add more features.  We do this every day.  When you get into the industry, you will be _required_ to do it.  

It is better getting used to it now, than in the real job.

Please, consider this. 👍 

## Extra help and tips

- Use Jest.  This is the industry standard.
- To test components, we usually test if the expected output was rendered (e.g. `expect(tree).toMatchSnapshot();` `expect(list).toContain(apple);`.
- Jest expectations: https://jestjs.io/docs/expect
- Jest for React main docs: https://jestjs.io/docs/tutorial-react
- Helpful article: https://www.browserstack.com/guide/unit-testing-of-react-apps-using-jest
- This is EASIER than it looks 

Please, kindly add some tests to ensure your app's integrity.  Also, this ensures you will be prepared for your upcoming job! 💪